app/src/main/java/com/jutaav/activity/WSLoadingActivity.kt

```diff
@@ -21,6 +22,10 @@ class WSLoadingActivity : BaseActivity<ActivityWSLoadingBinding>(R.layout.activi
         try {
             supportActionBar?.hide()
 
+            Glide.with(this)
```

Comment:
Use `GlideModule`, gives us the flexibility to customise request and better API.